.goreleaser.yaml

```
@@ -0,0 +1,38 @@
+builds:
+  - main: ./cmd/zync
+    id: zync
+    binary: zync
+    env:
+      - CGO_ENABLED=0
+    ldflags:
+      - -s -X github.com/brimdata/zync/cli.version={{ .Tag }}
```

Need the proper package name here.

And we should use `.Version` instead of `.Tag`. (I'm not sure why I switched from `.Version` to `.Tag` in brimdata/zed#3768 but it was a mistake.)

```suggestion
      - -s -X github.com/brimdata/zync/cmd/zync/version.version={{ .Version }}
```